src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -359,6 +437,9 @@ private static class RandomIO {
         // buffer for next bits
         private byte[] nextBuffer;
 
+        // check the variant
+        private Variant flag;
```

Comment:
The comment doesn't add value, and the name `flag` isn't very descriptive. The field should be `final`:
```java
        private final Variant variant;
```

Reply:
updated